openapi.yaml

```diff
@@ -957,6 +957,50 @@ paths:
       - lang: CLI
         source: >
           linode-cli account login-view 1234
+  /account/maintenance:
+    x-linode-cli-command: maintenance
+    get:
+      x-linode-grant: read_only
+      tags:
+      - Maintenance
+      summary: List Maintenance
+      description: >
+        Returns a collection of Maintenance objects for any entity
+        a user has permissions to view.
```

Comment:
Only linodes will be in this list for the first release. Should we call that out?

Reply:
I added a note to the description, please let me know if this is accurate/ok

Reply:
lgtm!